pkg/api/types.go

```diff
@@ -1202,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
+	// Fs represents a filesystem-based local volume
+	// +optional
+	Fs *LocalFsVolume
```

Comment:
It's hypothetical, as we don't have a detailed investigation or plan for block support.  But I think it is a cleaner API to clearly separate the two types of sources, and also makes future expansion easier.